lms/templates/problem.html

```diff
@@ -42,16 +42,41 @@ <h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="
     </span>
     % endif
     </div>
-    % if check_button:
-    <button class="check ${ check_button } btn-brand" data-checking="${ check_button_checking }" data-value="${ check_button }">
-        <span class="check-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span>
+    <button type="button" class="submit ${ submit_button } btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
```

Comment:
Why is ${ submit_button} being put as a class? This is a string that will be translated, so you can't rely on it being an English string (or any particular string).

I realize this was an existing pattern, but isn't correct, and we should take the time to fix it.